Buildings/Controls/OBC/ASHRAE/G36/AHUs/SingleZone/SetPoints/ReliefDamper.mo

```diff
@@ -0,0 +1,208 @@
+﻿within Buildings.Controls.OBC.ASHRAE.G36.AHUs.SingleZone.SetPoints;
+block ReliefDamper "Control of actuated relief  dampers without fans"
+  parameter Real minRelDamPos(min=0, max=1, unit="1") = 0.1
+    "Relief damper position that maintains a building pressure of 0.05 inchWC (12.44 Pa) while the system is at MinPosMin 
+    (i.e., the economizer damper is positioned to provide MinOA while the supply fan is at minimum speed)."
```

Comment:
Shorten the comment, and don't use the value of the parameter in the comment. Otherwise if a user changes the parameter value, the comment is wrong.